### PR TITLE
Loosen manageiq-password to < 2

### PR DESCRIPTION
**Pull request by Fryguy**
Part of https://github.com/ManageIQ/manageiq/pull/21178

### CODE REVIEW

**Comment by agrare** [comment]
LGTM @Fryguy  but looks like we'll have to merge this in lockstep with manageiq-gems-pending
```
Bundler could not find compatible versions for gem "manageiq-password":
  In Gemfile:
    manageiq-gems-pending was resolved to 0.1.0, which depends on
      manageiq-password (~> 0.3)

    manageiq-smartstate was resolved to 0.8.0, which depends on
      manageiq-password (~> 1.0)
```

**Comment by Fryguy** [comment]
Oh interesting.  Yeah, so I think all of these still be merge and release in lockstep

File: manageiq-smartstate.gemspec [hunk 1]
```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "binary_struct",      "~> 2.1"
   spec.add_dependency "iniparse"
   spec.add_dependency "linux_block_device", "~>0.2.1"
-  spec.add_dependency "manageiq-password",  "~>0.3"
+  spec.add_dependency "manageiq-password",  "~>1.0"
```
Comment (bdunne) [inline manageiq-smartstate.gemspec:29]: Can this be loosened to `< 2`?

**Comment by Fryguy** [comment]
@bdunne Updated

**Pull request merged**